src/Converter/Feature2Mesh.js

```diff
@@ -522,12 +533,13 @@ function createInstancedMesh(mesh, count, ptsIn) {
  * Convert a [Feature]{@link Feature} of type POINT to a Instanced meshes
  *
  * @param {Object} feature
+ * @param {Object} options - options controlling the conversion
```

Comment:
Update doc to something more specific explaining which options can be passed and how they "control" the conversion